Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -52,6 +54,33 @@ private bool PowerWiresPulsed
             }
         }
 
+        private bool _boltsDown;
+        private bool BoltsDown
+        {
+            get => _boltsDown;
+            set
+            {
+                _boltsDown = value;
+                EntitySystem.Get<AudioSystem>()
+                .PlayFromEntity(value ? "/Audio/machines/boltsdown.ogg" : "/Audio/machines/boltsup.ogg",
```

Comment:
Doing stuff like playing audio inside setters is a *very* poor idea. The audio should be played in the code actually modifying the bolts (or a helper method).